html/webappapis/system-state-and-capabilities/the-navigator-object/protocol-handler-fragment-nosw.https.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<!-- Use a non-UTF-8 encoding to see how the handler URL is parsed -->
+<meta charset=windows-1254>
+<meta name=timeout content=long>
+<title>registerProtocolHandler() and a handler with %s in the fragment (does not use a service worker)</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src=/service-workers/service-worker/resources/test-helpers.sub.js></script>
+<script>
+'use strict';
+promise_setup(async () => {
+  await test_driver.set_rph_registration_mode("autoaccept");
```

Comment:
This doesn't match what got standardized, right? There's a hyphen missing.

Reply:
Yeah, it's not correct as it is now.

Reply:
Unresolving this as it wasn't resolved.

Reply:
I've submitted a new PR with the hyphenated values, as stated in the spec. I recall that @domenic [suggested](https://github.com/whatwg/html/pull/8267#discussion_r964356157) to follow the [Web Platform  design principles ](https://w3ctag.github.io/design-principles/#casing-rules) if there is no imposition from the WebDriver spec.

However, reading again the Payments Confirmation Dialog spec that inspired this new command, it used a camelCase version. I've recently implemented a [refactoring](https://chromium-review.googlesource.com/c/chromium/src/+/4218519)  so that both features use the same Enumeration for their respective commands. 

I wonder whether we should go back to the camelCase version I initially propose. This obviously would require a new PR for the HTML spec, but I think it'd still worth the effort for the shake of coherence. Obviously it'd be also possible to change the Payments Confirmation Dialog spec.

Opinions ?

Reply:
I would slightly prefer to change the Payments Confirmation Dialog spec, but not strongly or in a blocking way.

Reply:
I've submitted a [PR#231](https://github.com/w3c/secure-payment-confirmation/pull/231) to see how the Payments Confirmation Dialog have to say.

Reply:
In the [WebDriver issue 1725](https://github.com/w3c/webdriver/issues/1725) We are trying to get an agreement between the WebDriver implementers on a principle about the syntax. The first comments of @jgraham went on the direction of using camelCase for enums. 

I'll wait some weeks and If nobody shows opposition, I'll follow @jgraham's suggestion and proceed to chance the Custom Handlers section of the HTML spec to adapt the extension command to the proposed syntax.

Reply:
Submitted a new commit using camelCase, as resolved in the issue that we have filed explicitly to address this.